api/nodes/serializers.py

```diff
@@ -150,13 +151,17 @@ def create(self, validated_data):
         auth = Auth(user)
         node = self.context['view'].get_node()
         pointer_node = Node.load(validated_data['node']['_id'])
-        pointer = node.add_pointer(pointer_node, auth, save=True)
-        return pointer
+        if not pointer_node:
+            raise exceptions.NotFound('Node not found.')
```

Comment:
For custom error messages, we'll probably want to use Django's standard methods for internationalization. 

See http://www.django-rest-framework.org/topics/internationalization/ and https://docs.djangoproject.com/en/1.8/topics/i18n/translation/. I believe this just involves adding a `ugettext` call around the messages, but I haven't looked too deeply into this. @brianjgeiger @pattisdr Can you look into this when you get a chance? It's not necessary to implement before this is merged, but supporting il8n early will save us time in the future.